Essentials/src/com/earth2me/essentials/signs/SignEnchant.java

```diff
@@ -28,21 +28,19 @@ protected boolean onSignCreate(final ISign sign, final User player, final String
             throw e;
         }
         final String[] enchantLevel = sign.getLine(2).split(":");
-        if (enchantLevel.length != 2) {
-            sign.setLine(2, "§c<enchant>");
-            throw new SignException(tl("invalidSignLine", 3));
-        }
+        int level = 1;
         final Enchantment enchantment = Enchantments.getByName(enchantLevel[0]);
         if (enchantment == null) {
             sign.setLine(2, "§c<enchant>");
             throw new SignException(tl("enchantmentNotFound"));
         }
-        int level;
-        try {
-            level = Integer.parseInt(enchantLevel[1]);
-        } catch (NumberFormatException ex) {
-            sign.setLine(2, "§c<enchant>");
-            throw new SignException(ex.getMessage(), ex);
+        if (enchantLevel.length == 2) {
```

Comment:
This logic opens up a weird validation corner case that probably doesn't matter but I'll still complain about....

`enchant:level:somejunk` is valid, but even if level > 1, level will always only be 1. This should be a validation error.